client/webserver/site/src/html/wallets.tmpl

```diff
@@ -721,8 +721,8 @@
           </tr>
         </tbody>
       </table>
-      <div class="d-flex justify-content-end">
-        <input id="addPeerInput">
+      <div class="d-flex justify-content-end mt-4">
+        <input id="addPeerInput" placeholder="[[[enter_peer_address]]]">
```

Comment:
Why is this input the wrong background color. Do we need `type="text"`?

Reply:
Oh yeah.